go-concourse/concourse/internal/connection.go

```diff
@@ -218,6 +218,8 @@ func (connection *connection) populateResponse(response *http.Response, returnRe
 		}
 	}
 
+	showPolicyCheckWarningIfHas(response)
```

Comment:
Side note, if `shouldBlock` is set to `true`, any fly commands (vast majority) that use the deprecated `Connection` interface will get a cryptic `error: forbidden` message since the actual body of the response is not captured in the returned error.

Any commands (most notably `fly set-pipeline`) that use the newer `HttpAgent` interface will capture the body and prints a helpful
```
error: forbidden: policy check failed:
 * blah blah blah
 ```

Reply:
Not requesting any changes here, just pointing out that some users might get insufficient error messages if they use certain fly commands.

Reply:
If you look at the screen shot I attached in "Note to Reviewers", `fly workers` also shows detailed policy error messages. I believe `Connection` interface also can show policy messages.